src/main/java/at/ac/tuwien/kr/alpha/solver/heuristics/BerkMin.java

```diff
@@ -50,29 +50,33 @@
 public class BerkMin implements BranchingHeuristic {
 	
 	public static final double DEFAULT_ACTIVITY = 0.0;
+	public static final int DEFAULT_SIGN_COUNTER = 0;
```

Comment:
The default value for sign counters was used in multiple places. I extracted it as constant to lessen the risk of introducing inconsistencies.